indico/legacy/pdfinterface/base.py

```diff
@@ -503,13 +503,14 @@ def _doNothing(canvas, doc):
 
 class DocTemplateWithTOC(SimpleDocTemplate):
 
-    def __init__(self, indexedFlowable, filename, firstPageNumber = 1, **kw ):
+    def __init__(self, indexedFlowable, filename, firstPageNumber=1, include_toc=False, **kw):
```

Comment:
fyi, this class is used in other places  where the TOC is now disabled too - so this argument should be True by default (i'm changing it, no need to update the PR)